.travis.yml

```diff
@@ -4,33 +4,15 @@ rust:
 # This represents the minimum Rust version supported by Tokio. Updating this
 # should be done in a dedicated PR and cannot be greater than two 0.x
 # releases prior to the current stable.
-- 1.26.0
+- 1.34.0
 - stable
 - beta
 - nightly
 
 script:
-- cargo test --verbose -p tokio-trace-futures
-- cargo test --verbose -p tokio-trace-subscriber
-- cargo test --verbose -p tokio-trace-log
-- cargo test --verbose -p tokio-trace-env-logger
-- cargo test --verbose -p tokio-trace-macros
-- cargo test --verbose -p tokio-trace-serde
-- cargo test --verbose -p tokio-trace-fmt
-- cargo test --verbose -p tokio-trace-slog
-
+- cargo test --all
```

Comment:
Should this keep `--verbose`? Do we also need to take features into account here? Testing with `--no-default-features` for example can be handy. Personally I also like doing a first `check` pass [like this](https://github.com/jonhoo/inferno/blob/994b61ff67862c7797d8bb944a057c7cf3852fe4/.travis.yml#L19-L21) to eliminate bad commits early.